.ci/validation.sh

```diff
@@ -168,7 +168,7 @@ no-error-xwiki)
   CS_POM_VERSION="$(getCheckstylePomVersion)"
   echo version:$CS_POM_VERSION
   mvn -e --no-transfer-progress clean install -Pno-validations
-  checkout_from "https://github.com/xwiki/xwiki-commons.git"
+  checkout_from "-b checkstyle_9280 https://github.com/checkstyle/xwiki-commons.git"
```

Comment:
please add comment `until http:/some-draft-pr-to-xwiki` to know when we are ok to return back to xwiki code validation, as usage of forked project (dead code) is not very beneficial.